addons/main/functions/UnitAction/fnc_doFleeing.sqf

```diff
@@ -94,12 +96,13 @@ if (_onFootAndSeen) then {
 
     // find buildings to hide
     private _buildings = [_unit, SEARCH_FOR_BUILDING, true, true] call FUNC(findBuildings);
-    if ((_buildings isNotEqualTo []) && {_distance2D > random 5}) then {
+    _buildings append (_cover apply {getPos _x});
```

Comment:
``getPos _x`` -> ``ASLtoAGL (getPosASL _x)``

Position format AGL is preferable for ``doMove``, and [getPos is a bad practice](https://community.bistudio.com/wiki/getPos)

Reply:
We'll review that change later. (We like it).

Reply:
We will be looking at that with the other pull request.